lib/services/references.js

```diff
@@ -24,7 +29,7 @@ function replaceVersion(uri, version) {
 
 /**
  * @param {string} version
- * @returns {Function}
+ * @returns {function}
```

Comment:
Ha, I believe it is truly an instance of Function with a capital F, but consistency with the other JSDocs is good.